docs/installation.md

```diff
@@ -0,0 +1,53 @@
+# Installation
+Pulse can be installed as a Cloudera CSD (Custom Service Descriptor). See [CSD](csd)
```

Comment:
CSD and PARCEL link to 404s (https://pulse-logging.readthedocs.io/en/2.1.10/csd)